libs/stokesian_dynamics/device_matrix.hpp

```diff
@@ -0,0 +1,658 @@
+// Copyright (c) 2018 Henri Menke. All rights reserved.
+
```

Comment:
Please add a high level comment explaining what this header provides

Reply:
Also, this license header is not compatible with the GNU GPL.